src/app/complex/complex-results/complex-results.component.html

```diff
@@ -1,41 +1,65 @@
   <ng-container *ngIf="complexSearch;else loadingSpinner">
-  <div class="columns medium-12" *ngIf="complexSearch.totalNumberOfResults !== 0">
-    <h2 class="padding-left-large">Total number of results: {{complexSearch.totalNumberOfResults}} </h2>
-    <div class="columns medium-4">
-      <cp-complex-filter *ngIf="spicesFilter && bioRoleFilter && interactorTypeFilter"
-                         [spicesFilter]="spicesFilter"
-                         [bioRoleFilter]="bioRoleFilter"
-                         [interactorTypeFilter]="interactorTypeFilter" [facets]="complexSearch.facets"
-                         (onResetAllFilters)="onResetAllFilters()"
-                         (onSpicesFilterChanged)="onSpicesFilterChanged($event)"
-                         (onBiologicalRoleFilterChanged)="onBiologicalRoleFilterChanged($event)"
-                         (onInteractorTyoeFilterChanged)="onInteractorTyoeFilterChanged($event)">
-      </cp-complex-filter>
+    <div class="columns medium-12" *ngIf="complexSearch.totalNumberOfResults !== 0">
+      <h2 class="padding-left-large">Total number of results: {{ complexSearch.totalNumberOfResults }} </h2>
+      <div class="row">
+        <div class="displayButton columns medium-8" style="text-align: center">
+          <input type="submit" name="submit" value="List View" class="button view-button"
+                 (click)="setListView()" [ngClass]="{'selected': DisplayType === LIST_VIEW}">
+          <input type="submit" name="submit" value="Complex Navigator View" class="button view-button"
+                 (click)="setComplexNavigatorView()" [ngClass]="{'selected': DisplayType === COMPLEX_NAVIGATOR_VIEW}">
+        </div>
+      </div>
+      <div class="columns medium-4">
+        <cp-complex-filter *ngIf="spicesFilter && bioRoleFilter && interactorTypeFilter"
+                           [spicesFilter]="spicesFilter"
+                           [bioRoleFilter]="bioRoleFilter"
+                           [interactorTypeFilter]="interactorTypeFilter" [facets]="complexSearch.facets"
+                           (onResetAllFilters)="onResetAllFilters()"
+                           (onSpicesFilterChanged)="onSpicesFilterChanged($event)"
+                           (onBiologicalRoleFilterChanged)="onBiologicalRoleFilterChanged($event)"
+                           (onInteractorTypeFilterChanged)="onInteractorTypeFilterChanged($event)">
+        </cp-complex-filter>
+      </div>
+      <div class="columns medium-8">
+        <cp-complex-paginator [currentPageIndex]="currentPageIndex"
+                              [lastPageIndex]="lastPageIndex"
+                              (onPageChange)="onPageChange($event)"></cp-complex-paginator>
+        <div class="listOfResults" *ngIf="DisplayType===LIST_VIEW">
+          <cp-complex-list [complexSearch]="complexSearch">
+          </cp-complex-list>
+        </div>
+        <div class="Complex-navigator" *ngIf="DisplayType===COMPLEX_NAVIGATOR_VIEW">
```

Comment:
I would change this to
```
            <cp-complex-navigator class="Complex-navigator" *ngIf="DisplayType===COMPLEX_NAVIGATOR_VIEW" 
                                  [complexSearch]="complexSearch"
                                  [interactors]="allInteractorsInComplexSearch">
            </cp-complex-navigator>
```

And move `interactorsSorting` variable to the complex navigator component and update the inputs. Then in the complex navigator html do
```
<div class="ComplexNavigator">
  <cp-complex-navigator-sorting
            [(interactorsSorting)]="interactorsSorting">
  </cp-complex-navigator-sorting>
  <div class="Matrix">
    <cp-table-structure [complexSearch]="complexSearch"
                        [interactors]="interactors"
                        [interactorsSorting]="interactorsSorting">
    </cp-table-structure>
  </div>
</div>
```